dspace/config/hibernate-ehcache-config.xml

```diff
@@ -0,0 +1,586 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    The contents of this file are subject to the license and copyright
+    detailed in the LICENSE and NOTICE files at the root of the source
+    tree and available online at
+
+    http://www.dspace.org/license/
+
+-->
+<ehcache xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="ehcache.xsd">
+
+    <!-- Sets the path to the directory where cache .data files are created.
+
+         If the path is a Java System Property it is replaced by
+         its value in the running VM.
+
+         The following properties are translated:
+         user.home - User's home directory
+         user.dir - User's current working directory
+         java.io.tmpdir - Default temp file path -->
+
+    <!-- WARNING: If you are running multiple DSpace instances on the same server, make sure to start
+         each DSpace instance with another value for java.io.tmpdir !!! -->
```

Comment:
How to do this?  It's a system property, and every Context in the container will have the same value for it.